packages/manager/modules/billing-components/src/components/subscription-tile/subscription-tile.html

```diff
@@ -43,20 +43,25 @@
             <div data-ng-if="!$ctrl.isLoading">
                 <span
                     class="oui-badge oui-badge_error"
-                    data-ng-if="!$ctrl.service.isEngaged()"
+                    data-ng-if="!$ctrl.service.isEngaged() && !$ctrl.serviceInfos.hasPendingEngagement"
                     data-translate="manager_billing_subscription_engagement_status_none"
                 ></span>
                 <span
-                    data-ng-if="$ctrl.service.isEngaged() && !$ctrl.service.isEngagementExpired()"
-                    data-translate="manager_billing_subscription_engagement_status_engaged"
+                    data-ng-if="$ctrl.service.isEngaged() && !$ctrl.service.isEngagementExpired() && !$ctrl.serviceInfos.hasPendingEngagement"
+                    data-translate="{{ $ctrl.serviceInfos.engagementDetails.endRule.strategy !== $ctrl.AUTO_COMMITMENT_STRATEGY ? 'manager_billing_subscription_engagement_status_engaged_expired' : 'manager_billing_subscription_engagement_status_engaged_renew' }}"
```

Comment:
I know it is for translate values purpose, but may the test be moved to instance? (same as for `hasPendingEngagement`?